File: src/main/java/com/cloudpi/cloudpi_backend/files/structure/services/FileSystemInfoService.java
package com.cloudpi.cloudpi_backend.files.structure.services;

import com.cloudpi.cloudpi_backend.files.structure.dto.DirectoryDto;
import com.cloudpi.cloudpi_backend.files.structure.dto.FileDto;

public interface FileSystemInfoService {

    FileDto getFile(String path);

    DirectoryDto getDirectory(String path);

    void createNewFile(String path, Long size);

    void deleteFile(String path);
}

File: src/main/java/com/cloudpi/cloudpi_backend/files_info/services/FileSystemInfoService.kt
package com.cloudpi.cloudpi_backend.files_info.services;

import com.cloudpi.cloudpi_backend.files_info.dto.DirectoryDto;
import com.cloudpi.cloudpi_backend.files_info.dto.FileDto;

public interface FileSystemInfoService {

    FileDto getFile(String path);

    DirectoryDto getDirectory(String path);

    void createNewFile(String path, Long size);

    void deleteFile(String path);
}